_build/partials/sandbox_collectionsList.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="col-md-6 col-md-offset-1 datasets">
+  <div class="row">
+    {{#each datasets}}
+    <div id="{{Slug}}" class="sandbox-dataset">
+      <h4><a href="{{rootUrl}}{{Slug}}/sandbox-data.html" class="sandbox-dataset-title">{{Name}} Sandbox Data</a></h4>
+      
+      <div class="image-card">
+        {{#if Image}}
+          <div class="imagette">
+            <a href="{{rootUrl}}{{Slug}}/sandbox-data.html">
+              <img style="border: solid 0.5px lightgray" src="{{rootUrl}}{{Image}}" alt="{{Slug}}" />
+            </a>
+          </div>
+        {{/if}}
+        <div class="info">
+          <p>{{{trimHTML (md Description) 500}}}</p>
+          <p><a href="{{rootUrl}}{{Slug}}/sandbox-data.html">Sandbox data for {{Name}} &rarr;</a></p>
+        </div>
+
+      </div>
+      
```

Comment:
Check and remove empty lines where not needed ;)

Reply:
should be fixed in https://github.com/sentinel-hub/public-collections/pull/114